frontend/lang/messages/en-US.json

```diff
@@ -1271,7 +1271,9 @@
     "debug-openai-services": "Debug OpenAI Services",
     "debug-openai-services-description": "Use this page to debug OpenAI services. You can test your OpenAI connection and see the results here. If you have image services enabled, you can also provide an image.",
     "run-test": "Run Test",
-    "test-results": "Test Results"
+    "test-results": "Test Results",
+    "group-delete-note": "Groups with users or households cannot be deleted.",
```

Comment:
Should we remove the periods? I think short tooltips typically don't have them

Reply:
Yeah, i agree 👍